### ISSUE TITLE
RSW: prestart-launcher.bash prevents use of K8s templates mounted via projected volume

### ISSUE THREAD

**Issue by jgoldschrafe** [rstudio/helm#274]
Currently, the RStudio Workbench pod performs a startup initialization which attempts to enforce specific ownership on `/var/lib/rstudio-launcher/Kubernetes` and its children:

```bash
      mkdir -p \
        /var/lib/rstudio-launcher/Local \
        /var/lib/rstudio-launcher/Kubernetes
      chown -v -R \
        rstudio-server:rstudio-server \
        /var/lib/rstudio-launcher 2>&1 | _indent
```

Unfortunately, this startup task prevents administrators from mounting these templates via projected volume (such as ConfigMap), because the ownership of a ConfigMap is determined by the pod/container `securityPolicy`. By default, this will be `root:root`, unless overridden.

When using the default ownership, RStudio Workbench produces this error:

```
#----> prestart-launcher.bash 2022-09-06T20:48:34: Preparing dirs
       chown: changing ownership of '/var/lib/rstudio-launcher/Kubernetes/pod.tpl': Read-only file system
       failed to change ownership of '/var/lib/rstudio-launcher/Kubernetes/pod.tpl' from root:root to rstudio-server:rstudio-server
       chown: changing ownership of '/var/lib/rstudio-launcher/Kubernetes/service.tpl': Read-only file system
       failed to change ownership of '/var/lib/rstudio-launcher/Kubernetes/service.tpl' from root:root to rstudio-server:rstudio-server
       ownership of '/var/lib/rstudio-launcher/Kubernetes' retained as rstudio-server:rstudio-server
       ownership of '/var/lib/rstudio-launcher/Local' retained as rstudio-server:rstudio-server
       ownership of '/var/lib/rstudio-launcher' retained as rstudio-server:rstudio-server
```

The pod security policy can be used to set the group via `fsGroup`. However, there is no corresponding `fsUser` property, only `runAsUser`. Attempting to `runAsUser: 999` results in the following error instead, as the launcher (or its process supervisor) expects to be started as root and then drop privileges itself:

```
2022-09-06 20:47:55,758 CRIT Can't drop privilege as nonroot user
2022-09-06 20:47:55,758 CRIT Can't drop privilege as nonroot user
```

**Comment by jgoldschrafe** [comment]
Very probably duplicative with #213, which seems to have also encountered (and worked around) this issue quite recently. Since that seems to be taking a long time to get over the finish line, it might still be worth tracking this issue separately.

**Comment by azariaharendt** [comment]
Closed in #278

**Comment by colearendt** [comment]
Closed in #278
